package.json

```diff
@@ -43,27 +43,26 @@
   "dependencies": {
     "@babel/core": "^7.4.4",
     "@sentry/browser": "^5.2.0",
-    "@storybook/addon-a11y": "3.4.11",
-    "@storybook/addon-actions": "3.4.11",
-    "@storybook/addon-backgrounds": "3.4.11",
-    "@storybook/addon-info": "3.4.11",
-    "@storybook/addon-knobs": "3.4.11",
-    "@storybook/addon-options": "3.4.11",
-    "@storybook/addon-storyshots": "3.4.11",
-    "@storybook/addon-viewport": "3.4.11",
-    "@storybook/addons": "3.4.11",
-    "@storybook/react": "3.4.11",
+    "@storybook/addon-a11y": "5.0.11",
+    "@storybook/addon-actions": "5.0.11",
+    "@storybook/addon-backgrounds": "5.0.11",
+    "@storybook/addon-info": "5.0.11",
+    "@storybook/addon-knobs": "5.0.11",
+    "@storybook/addon-storyshots": "5.0.11",
+    "@storybook/addon-viewport": "5.0.11",
```

Comment:
Can you add carets before the version numbers for the Storybook dependencies?

Reply:
Done!